### PR TITLE
[ADD] signature page

### PR DESCRIPTION
**Pull request by OdakK**

### CODE REVIEW

**Comment by OdakK** [comment]
[Pages de signature](https://trello.com/c/RWUyes9n/103-pages-de-signature)

**Review by LfrPaul — APPROVED** [review]

File: app/src/main/java/com/immobylette/appmobile/MainActivity.kt [hunk 1]
```diff
@@ -83,7 +89,7 @@ class MainActivity : ComponentActivity() {
                         propertySelectionViewModel = propertySelectionViewModel,
                         currentPropertyViewModel = currentPropertyViewModel,
                         onNavigateToChangeAgent = navController::navigateToAgentSelection,
-                        onNavigateToPropertySelected = navController::navigateToConfirmation
+                        onNavigateToPropertySelected = navController::navigateToAgentSignature
```
Comment (patacoing) [inline app/src/main/java/com/immobylette/appmobile/MainActivity.kt]: Tu as oublié de l'enlever

File: .idea/misc.xml
Comment (patacoing) [inline .idea/misc.xml:1]: supp

File: .idea/deploymentTargetSelector.xml
Comment (patacoing) [inline .idea/deploymentTargetSelector.xml:1]: supp

**Review by patacoing — CHANGES_REQUESTED** [review]

**Review by patacoing — APPROVED** [review]

**Pull request merged**